api/client/trust.go

```diff
@@ -425,6 +425,10 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	}
 
 	err = repo.Publish()
+	if err == nil {
+		fmt.Fprintf(cli.out, "Finished signing %q with tag %s\n", repoInfo.FullName(), tag)
```

Comment:
Trust as a whole is only run if a tag is specified at push time. This seems like exactly what we'd want: only print this "Finished ..." message if signing actually happened (which will only be when a tag is provided).

I'm in support of a success message of some kind. Rather than just "Finished signing ..." I might say "Successfully signed ..." Not going to hold things up on that though. It's fine as is too.